xbmc/platform/android/jni/AudioFormat.cpp

```diff
@@ -84,6 +86,10 @@ void CJNIAudioFormat::PopulateStaticFields()
         {
           CJNIAudioFormat::ENCODING_DTS = get_static_field<int>(c, "ENCODING_DTS");
           CJNIAudioFormat::ENCODING_DTS_HD = get_static_field<int>(c, "ENCODING_DTS_HD");
+          // Nvidia Shield v6 firmware uses another ID, which will also be the future ID
+          // though other v23 version would not return a value if we'd use the get_static_field
+          // method to query this value. The hardcoded value can be removed after probably is out
+          CJNIAudioFormat::ENCODING_DOLBY_TRUEHD = 0x0000000d;
```

Comment:
See https://github.com/koying/xbmc/blob/master/xbmc/android/jni/AudioFormat.cpp#L97 and https://github.com/koying/xbmc/blob/master/xbmc/android/jni/AudioFormat.cpp#L57 to get cross-api values while taking into account the fact that the value might not be there